pkg/chain/ethereum/ethereum.go

```diff
@@ -405,6 +405,27 @@ func (ec *ethereumChain) ReportRelayEntryTimeout() error {
 	return nil
 }
 
+func (ec *ethereumChain) IsEntryInProgress() (bool, error) {
+	return ec.keepRandomBeaconOperatorContract.IsEntryInProgress()
+}
+
+func (ec *ethereumChain) CurrentEntryStartBlock() (*big.Int, error) {
```

Comment:
I just realized we have `currentRequest*` for all fields but this one and it's my fault because I did not notice it when refactoring `KeepRandomBeaconOperator` contract. Can we take the opportunity and rename `currentEntryStartBlock` to `currentRequestStartBlock` in the contract?

Reply:
just to make sure, we want `currentRequestStartBlock` not `currentRequestEntryStartBlock`?

Reply:
Yes, `currentRequestStartBlock`.